trunk/Admin/js/src/partials/Actions.js

```diff
@@ -41,14 +41,36 @@ let AdminQuickbarActions = {
 
     jQuery.post(aqbLocalize.ajaxUrl, {
       action: 'swift_performance_single_clear_cache',
-      '_wpnonce': $target.closest('.admin-quickbar, .admin-quickbar-jumpicons').data('swift-nonce'),
+      '_wpnonce': $target.closest('.admin-quickbar').data('swift-nonce'),
       'url': url,
     }, function(response) {
       $target.removeClass('is-in-cache');
       AdminQuickbarActions.addPageToSwiftCache(e);
     });
   },
 
+
+  /**
+   * Removes all pages from swift cache
+   *
+   * @param e
```

Comment:
I would suggest adding the Type to the Jsdoc Intelisense smarter.

```suggestion
   * @param {Event} e
```